### PR TITLE
RD-11468 - add motor tests and payload to mongo

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

File: src/test/integration/pymongo/tests/test_pymongo.py [hunk 1]
```diff
@@ -17,23 +17,30 @@ def test_mongo_instrumentation(self):
...
+            for s in SpansContainer.parse_spans_from_file().spans
+            if s["attributes"].get("db.system") == "mongodb"
+        ]
+
+        set_span, get_span = spans[
+            -2:
+        ]  # there are more spans to connect to the mongo container
 
         # assert mongo children spans
-        assert spans_container.find_child_span(
```
Comment (saartochner) [inline src/test/integration/pymongo/tests/test_pymongo.py:23]: `spans_container.find_child_span` returned a generator, so this line didn't check anything...

File: src/lumigo_opentelemetry/instrumentations/pymongo/__init__.py [hunk 2]
```diff
@@ -10,8 +17,43 @@ def check_if_applicable(self) -> None:
...
+            CommandSucceededEvent,
+            CommandFailedEvent,
+        )
+
+        def request_hook(span: Span, event: CommandStartedEvent) -> None:
+            with lumigo_safe_execute("pymongo_request_hook"):
+                span.set_attribute("db.statement", event.command_name)
+                if isinstance(event, CommandStartedEvent):
+                    span.set_attribute(
+                        "db.request.body",
```
Comment (therightstuff) [inline src/lumigo_opentelemetry/instrumentations/pymongo/__init__.py:31]: is this standard? i feel like db statements should be entirely in the statement itself, as it is the command seems to usually be the name of the span

**Review by therightstuff — APPROVED** [review]
looking great, i have one question about consistency with the opentelemetry conventions but i'm pre-approving 👍

File: src/lumigo_opentelemetry/instrumentations/pymongo/__init__.py (hunk 2 above)
Comment (saartochner) [reply src/lumigo_opentelemetry/instrumentations/pymongo/__init__.py:31]: This is the convention the we invent in order to standardized the API of DB instrumentations. Just like `db.response.body` in redis :)

**Pull request merged**